src/api/nodeRpcCall.js

```diff
@@ -1574,6 +1574,37 @@ const stakingBondExtra = async (value, walletAddress, callback) => {
   });
 }
 
+const getCongressMembers = async () => {
+  const api = await getApi();
+  const members = await api.query.council.members();
+  return members.map(m => m.toString());
```

Comment:
i believe toHoman will do the same

Reply:
There are no guarantees on `.toHuman()`. It's output is lossy (strips most enums etc.) and can change with update or with different Locale. There're also typing issues: toHuman returns `AnyJson`, while this will return `Array<String>`, which will be PITA if we ever decide to start using TypeScript.
If you are lazy, you can use `.toJSON()`. While it still returns `AnyJson`, at least it's defined how it will serialize each type.
Ideally, `.toHuman()` should be only used for debugging and `.toJSON()` only for serialization.

See also this comment by one of devs: https://tinyurl.com/5743yvm2